softdevice_controller/CHANGELOG.rst

```diff
@@ -9,6 +9,23 @@ Changelog
 
 All the notable changes to this project are documented on this page.
 
+nRF Connect SDK v2.5.2
+**********************
+
+All the notable changes included in the |NCS| v2.5.2 release are documented in this section.
+
+Bug fixes
+=========
+
+* Fixed an issue where the controller stopped generating advertising reports.
+  This could happen when the controller was running an extended cooperative scanner together with other activities, such as advertising or connection,
+  while receiving data in an extended advertising event that used ``AUX_CHAIN_IND``. (DRGN-21020)
+* Fixed an issue where the controller would stop sending ACL data packets to the host when controller to host flow control was enabled.
+  This could happen when a disconnection occurred before the host had issued the Host Number of Complete Packets command for the remaining ACL data packets.
+  Now the controller waits until after all ACL data packets have been acknowledged by the host before raising the Disconnection Complete event.
```

Comment:
```suggestion
  Now, the controller waits until all ACL data packets have been acknowledged by the host before raising the Disconnection Complete event.
```

Reply:
One comment: these sentences have already been reviewed by tech writer when we updated this changelog in our internal repo. Then I generate the nrfxlib changelog automatically to this PR. If I updates in this PR, then I have to back port the changes to our internal repo. My question is: is it expected to make 2 rounds doc review?

Reply:
It is always possible that the tech writer may miss something when reviewing the internal repo, thus two rounds is often a good idea.